src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,116 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+import java.util.Objects;
+
 public class ArrayList<T> implements List<T> {
+    private static final int CAPACITY = 10;
```

Comment:
Clear naming is important. This is default value and not representing current capacity, so DEFAULT_CAPACITY suits better.